.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
 name: continuous-integration
 
 env:
-  NUSHELL_CARGO_TARGET: ci
+  NUSHELL_CARGO_PROFILE: ci
```

Comment:
Can we make this a separate PR as it's unrelated to your change?

Reply:
I'm not sure that it's unrelated; there is a need to use a variable for the build target, and a variable for the build profile. Without this change, the build system would be using `CARGO_BUILD_TARGET` to specify the build target and `NUSHELL_CARGO_TARGET` to specify the build profile, which is pretty confusing. I'll change it back if you really want though.